Lib9c/Extensions/DecimalExtensions.cs

```diff
@@ -0,0 +1,22 @@
+using System.Globalization;
+using System.Numerics;
+
+namespace Nekoyume.Extensions
+{
+    public static class DecimalExtensions
+    {
+        public static (BigInteger majorUnit, BigInteger minorUnit) ToMajorAndMinorUnits(
+            this decimal value)
+        {
+            var strings = value.ToString(CultureInfo.InvariantCulture).Split('.');
+            var majorUnit = decimal.Parse(strings[0], CultureInfo.InvariantCulture);
+            if (strings.Length == 1)
+            {
+                return (new BigInteger(majorUnit), BigInteger.Zero);
+            }
+
+            var minorUnit = decimal.Parse(strings[1], CultureInfo.InvariantCulture);
+            return (new BigInteger(majorUnit), new BigInteger(minorUnit));
+        }
+    }
```

Comment:
Major/Minor unit concept seems too specific concepts about Libplanet's FAV. thus how about returning `FungibleAssetValue` with `Currency` here?

```suggestion
        public static FungibleAssetValue ToFungibleAssetValue(this decimal value, Currency currency)
        {
            byte decimalPlaces = currency.decimalPlaces;
            decimal rounded = Math.Round(value, decimalPlaces, MidpointRounding.ToZero); // or need to receive MidpointRounding instead.
            BigInteger majorUnit = (BigInteger)Math.Floor(rounded);
            BigInteger minorUnit = (BigInteger)Math.Floor((rounded - majorUnit) * (decimal)BigInteger.Pow(10, decimalPlaces));

            return new FungibleAssetValue(currency, majorUnit, minorUnit);
        }
   }
```

Reply:
I've fix some code where to use this extension method and it can be remove.